mbed-coap/sn_config.h

```diff
@@ -98,6 +98,11 @@
  * \brief Sets the CoAP re-send interval in seconds.
  * By default is 10 seconds.
  */
+
+#ifdef MBED_CONF_MBED_CLIENT_DEFAULT_RESPONSE_TIMEOUT
+#define DEFAULT_RESPONSE_TIMEOUT
```

Comment:
Value is missing

Reply:
doh..fixed :-)

Reply:
Should we also increase some of the defaults here?

Reply:
Client is already changing that to 5 seconds at runtime so let's keep it as it is.